code/_onclick/hud/human.dm

```diff
@@ -34,14 +34,17 @@
 	if(hud_data.icon)
 		ui_style = hud_data.icon
 
-	src.adding = list()
-	src.other = list()
+	adding = list()
```

Comment:
any particular reason for this change?

src. is usually unnecessary but sometimes it can be useful, as a signal to the compiler or the programmer, which context is meant